integration_tests/base_routes.py

```diff
@@ -493,4 +518,6 @@ async def async_raise():
         index_file="index.html",
     )
     app.startup_handler(startup_handler)
+    app.add_view("/sync_view", SyncView)
+    app.add_view("/async_view", AsyncView)
```

Comment:
Can we use the same convention as in the other routes for the tests? i.e. `/sync/view` instead of `/sync_view`. Maybe `/sync/view/decorator` for the decorated routes. This helps keep things consistent and organized :slightly_smiling_face:

Reply:
Great shout. Thank you! 😄